com.jetbrains.RustRover.yml

```diff
@@ -0,0 +1,139 @@
+app-id: com.jetbrains.RustRover
+command: rustrover
+runtime: org.freedesktop.Sdk
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+tags:
+  - proprietary
+
+finish-args:
+  - --allow=devel
+  - --device=all
+  - --env=RUST_ROVER_PROPERTIES=/app/bin/idea.properties
+  - --filesystem=host
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  - --filesystem=xdg-run/docker
+  - --filesystem=xdg-run/gnupg
+  - --filesystem=xdg-run/keyring
+  - --filesystem=xdg-run/podman
+  - --share=ipc
+  - --share=network
+  - --socket=gpg-agent
+  - --socket=ssh-auth
+  - --socket=x11
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
does this use libnotify for notifications? then this is not needed.

Reply:
as it's Java based cross-platform application I'm not sure if that's the case. would rather keep it here for safety